zaas-service/src/main/java/org/zowe/apiml/zaas/zaas/ZaasController.java

```diff
@@ -45,7 +45,7 @@
 @RestController
 @RequestMapping(ZaasController.CONTROLLER_PATH)
 public class ZaasController {
-    public static final String CONTROLLER_PATH = "/zaas/zaas"; // FIXME: solve the structural issue
+    public static final String CONTROLLER_PATH = "/zaas/scheme"; // NOSONAR
```

Comment:
what's the sonar warning here?

Reply:
I'm not sure about that, maybe @arxioly can clarify